taegis_magic/core/normalizer.py

```diff
@@ -67,6 +67,26 @@ def validate_int(value: int) -> Union[int, str]:
         return template.render(obj=self)
 
 
+class TaegisResult(TaegisResultsNormalizer):
+    """Generic single result normalizer."""
+
+    raw_results: Any = field(default=None)
```

Comment:
Is there is equivalent of `Any` for any dataclass? Since the type here needs to work with `asdict` it isn't really `Any`.

Reply:
Going to leave this here for others.

`asdict` does handle most types, I'm sure we'll eventually find something that breaks it, but putting these here for reference on how it works:

https://github.com/python/cpython/blob/main/Lib/dataclasses.py#L227
https://github.com/python/cpython/blob/main/Lib/dataclasses.py#L1334